### DISCUSSION TITLE
What does the name "Feishin" mean?

### DISCUSSION THREAD

**Discussion by wayneyaoo** [jeffvli/feishin#115] [General]
Just curious ... Asked GPT it doesn't know what languages it originates : )

**Comment by jeffvli** [comment]
Hey there -

It originates from the Japanese serialized manga [Kingdom](https://en.wikipedia.org/wiki/Kingdom_(manga)). Kingdom takes place in the Warring States period in China and follows an army called the Fēi Xìn (Hi Shin in Japanese). With a bit of English normalization, I decided to call it Feishin.

So in summary -- it's a Chinese name inspired inspired from a Japanese manga and has nothing to do music.

**Reply by wayneyaoo** [reply]
Interesting.. when I first pronounced "feishin" it did sound like FeiXin (飞信?) but no idea what that means. so it turns out to be ... the name of a war unit in a virtual story in Kingdom.. Thanks for the info!

**Comment by ghost** [comment]
> Hey there -
> 
> It originates from the Japanese serialized manga [Kingdom](https://en.wikipedia.org/wiki/Kingdom_(manga)). Kingdom takes place in the Warring States period in China and follows an army called the Fēi Xìn (Hi Shin in Japanese). With a bit of English normalization, I decided to call it Feishin.
> 
> So in summary -- it's a Chinese name inspired inspired from a Japanese manga and has nothing to do music.

Sorry to necro a thread (is that a thing on GitHub? Is this technically a forum of sorts?), but I have to ask since I'm not familiar with this manga... is it pronounced `fey-shinn` or `fiy-shinn`? It's an awesome program, and I want to make sure I'm pronouncing it correctly :)

**Reply by jeffvli** [reply]
`fey-shinn` would be the correct one!
